### ISSUE TITLE
ssd-mobilenetv2 takes some time.

### ISSUE THREAD

**Issue by DigeSegun** [dusty-nv/jetson-inference#1644]
@dusty-nv Every time I run detectnet it takes forever to start running because I think ssd-mobilenetv2 is trying to load. Even though I have ran the script many times it still takes forever to load. Is there a reason or fix to get out of the long wait time

**Comment by dusty-nv** [comment]
Hi @DigeSegun, the first time a new model is loaded (or when that model has changed), TensorRT optimizes the model and saves the optimized engine to disk.   It should then start up quickly the next time.   After the program runs, does it save a .engine file in your model's folder?

**Comment by DigeSegun** [comment]
@dusty-nv I don't believe it saves an .engine file. How long goes optimization usually take I've ran the script all the way through multiple times

**Comment by DigeSegun** [comment]
@dusty-nv [TRT]    network profiling complete, saving engine cache to /usr/local/bin/networks/SSD-Mobilenet-v2/ssd_mobilenet_v2_coco.uff.1.1.8201.GPU.FP16.engine

[TRT]    failed to open engine cache file for writing /usr/local/bin/networks/SSD-Mobilenet-v2/ssd_mobilenet_v2_coco.uff.1.1.8201.GPU.FP16.engine

**Comment by dusty-nv** [comment]
> [TRT] failed to open engine cache file for writing /usr/local/bin/networks/SSD-Mobilenet-v2/ssd_mobilenet_v2_coco.uff.1.1.8201.GPU.FP16.engine

Are you running the jetson-inference container?   If not, did you run `sudo make install` when you built jetson-inference from source?

You might want to try:   `sudo chown -R $USER /usr/local/bin/networks`

**Comment by DigeSegun** [comment]
@dusty-nv I am running from source and I believe I did do a sudo make install. I just tried the sudo chown command and this is my output now 

[TRT]    failed to open engine cache file for writing /usr/local/bin/networks/SSD-Mobilenet-v2/ssd_mobilenet_v2_coco.uff.1.1.8201.GPU.FP16.engine
[TRT]    device GPU, completed saving engine cache to /usr/local/bin/networks/SSD-Mobilenet-v2/ssd_mobilenet_v2_coco.uff.1.1.8201.GPU.FP16.engine
[TRT]    saving model checksum to /usr/local/bin/networks/SSD-Mobilenet-v2/ssd_mobilenet_v2_coco.uff.sha256sum
[TRT]    sha256sum /usr/local/bin/networks/SSD-Mobilenet-v2/ssd_mobilenet_v2_coco.uff | awk '{print $1}' > /usr/local/bin/networks/SSD-Mobilenet-v2/ssd_mobilenet_v2_coco.uff.sha256sum
[TRT]    failed to save model checksum to /usr/local/bin/networks/SSD-Mobilenet-v2/ssd_mobilenet_v2_coco.uff.sha256sum
[TRT]    device GPU, loaded /usr/local/bin/networks/SSD-Mobilenet-v2/ssd_mobilenet_v2_coco.uff
